### ISSUE TITLE
Widget button does not appear

### ISSUE THREAD

**Issue by iSupersniper** [MALSync/MALSync#1237]
### Describe the Bug

When uBlock Origin is enabled the widget button does not appear.  When checking the logs it also does not seem to appear either.  

If I disable uBlock Origin, the widget button appears and shows up in the logs when the widget menu is opened.

I cannot figure out why it is getting blocked by uBlock Origin.  I have noticed this exact same problem with [All the anime top buttons](https://greasyfork.org/en/scripts/404760-all-the-anime-top-buttons) userscript, however, the [Copy Video Source Link](https://greasyfork.org/en/scripts/439879-copy-video-source-link) userscript widget button shows up just fine.

In the images below the left widget, icon is for the copy video source link
On the right, the arrow above the malsync icon is the all the anime top buttons icon.

Does anyone have a solution?

uBlock Origin Disabled
![ubo_disabled](https://user-images.githubusercontent.com/17890753/180118987-7d73f23c-ecfb-413e-a14a-9162a2c202ec.PNG)

uBlock Origin Enabled
![ubo_enabled](https://user-images.githubusercontent.com/17890753/180118990-bc259ded-af71-43fb-a158-35ca7138dbff.PNG)

### Steps to reproduce the behavior

Enable uBlock Origin and the widget icon does not appear
Disable uBlock Origin and the widget icon does appear 

### Browser

Microsoft Edge Chromium Version 103.0.1264.62 (Official build) (64-bit)

### Type

Userscript

### URL

https://github.com/MALSync/MALSync/releases/latest/download/malsync.user.js

**Comment by iSupersniper** [comment]
So an update

I reset the extension to factory default and am only using the default filterlists and this problem is gone.

It seems the problem lies in the custom filters or filter lists that I have.

**Comment by lolamtisch** [comment]
Normally it is because of whitelisting adblocking, but dont have much idea about filter lists. But either way as long it works with the default filter list it is out of scope for support.

**Comment by Karmesinrot** [comment]
Changing the injection mode of the userscript manager normally avoids this problem. If you are using tampermonkey you can't do that but uBO doesn't affect it for a few years now.

**Comment by iSupersniper** [comment]
Well I know you said because it works with all the default scripts that you didn't want to support it, however, I found that only 1 custom filter list was the culprit.

[Fuck Fuckadblock](https://github.com/bogachenko/fuckfuckadblock) was the culprit that blocked your widget button from showing up.
